automated install/basic-install.sh

```diff
@@ -138,8 +145,68 @@ show_ascii_berry() {
 }
 
 # Compatibility
+
+53probe (){
+  # Probe localhost via 127.0.0.1 for open port 53 and store process name if port 53 is occupied
+  if (echo > /dev/tcp/127.0.0.1/53) >/dev/null 2>&1; then
+    # What process is using 53?
+    who53="$(${SUDO} lsof -i :53 +c 0 | awk 'FNR==2{ print $1 }')"
+  fi
+}
+
+53check(){
+  # Print info regarding probing for open port 53
+  echo -e "  ${INFO} Testing for port 53 availability...${COL_NC}"
+  53probe
+  # Check running process and see if it's blank
+  if [ -z "$who53" ]; then
+    echo -e "  ${TICK} ${COL_LIGHT_GREEN}Port 53 is available.${COL_NC}"
+    # Check running process and see if it's pihole-FTL
+  elif [ "$who53" = "pihole-FTL" ]; then
+    # Proceed with install
+    echo -e "  ${TICK} Port 53 is in use by our resolver ${COL_LIGHT_GREEN}($who53)${COL_NC}, proceeding with setup"
+  elif [ "$who53" = "dnsmasq" ]; then
+    # If dnsmasq is present, set the dnsmasq-flag to true for future reference
+    # (after packages and dependencies are installed).
+    # dnsmasq will be disabled at end of install, prior to FTLDNS start.
+    echo -e "  ${EXCL} Port 53 is in use by ${COL_LIGHT_RED}$who53${COL_NC}. 
```

Comment:
Would prefer an opt-out/opt-in in the cases where the installer is going to do an action to a users system like disabling a system daemon.

Reply:
Would adding a flag to disable the behavior work?

Reply:
How would a flag work on an install? Would we have to tell people now `curl install.pihole.net | bash --no-clobber`? We'd have to update a ton of files and places to let people know, and I'd rather be opt in than opt out.